web-api-shared/src/elastic.rs

```diff
@@ -304,16 +440,39 @@ impl Client {
         })
     }
 
-    pub async fn query_with_bytes<B, T>(
+    pub async fn query_with_bytes<T>(
+        &self,
+        method: Method,
+        url: Url,
+        post_data: Option<(HeaderMap<HeaderValue>, Bytes)>,
```

Comment:
We change this to `Bytes` to make sure cloning the body between retries is cheap.